Patches/Core/ThingDefs_Misc/Weapons_Guns.xml

```diff
@@ -814,6 +814,7 @@
       <ammoSet>AmmoSet_556x45mmNATO</ammoSet>
     </AmmoUser>
     <FireModes>
+      <aimedBurstShotCount>5</aimedBurstShotCount>
```

Comment:
Auto turrets don't get burst fire modes--if they're capable of full auto, that's what they use. This inaccuracy and ammo consumption is an intended drawback of auto turrets vs manned ones.